VideoWeb/VideoWeb.UnitTests/Common/Models/GetNonScreenedParticipantsAndEndpointsTests.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using VideoWeb.Common.Models;
+using VideoWeb.UnitTests.Builders;
+
+namespace VideoWeb.UnitTests.Common.Models;
+
+public class GetNonScreenedParticipantsAndEndpointsTests
```

Comment:
there is already a test class for this:

VideoWeb.UnitTests.Models.Conference.GetNonScreenedParticipantsAndEndpointsTests

could they be merged together?

Reply:
will do, I couldn't find it under usages